docs/agent/config/v2.mdx

```diff
@@ -1,43 +1,11 @@
 ---
-title: Configuration File
+title: "Version 2"
```

Comment:
This is "almost" a 1-1 copy of the previous configuration file, but now instead under v2